src/Operator.ts

```diff
@@ -1,7 +1,7 @@
 import {Subscriber} from './Subscriber';
 
 export class Operator<T, R> {
-  call<T, R>(subscriber: Subscriber<R>): Subscriber<T> {
+  call<R>(subscriber: Subscriber<R>): Subscriber<T> {
```

Comment:
In this case, the type `R` of `call` does not have any relation with `R` of `Operator`?